modules/ocf_mail/manifests/firewall_input.pp

```diff
@@ -0,0 +1,20 @@
+# firewall input rules for ocf_mail, allow submission (587), allow smtp (25)
```

Comment:
Not sure this needs to be a separate file - it's not a separate functional module (unlike `spam`, `site_vhost`, ...). Consider placing these rules in `init.pp`?